summary.r

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env Rscript
+# Author: OLP/2022
+# Usage: summary for SARS-CoV-2 run, only for PatientSamples, input: one parameter
+# ie. Rscript /. . . ./summary.r UT-A01290-220416
+
+library("plyr", warn.conflicts = F)
+library("dplyr", warn.conflicts = F)
+library("rbin")#
+library("tidyverse")
+library("lubridate", warn.conflicts = F)
+library("tibble")
+library("stringr") 
+require("seqinr"); 
+library("XML")
+#===============================================
+args = commandArgs(trailingOnly=T)
+# args[1]<-c('UT-A01290-220416'); # only to works on script preparation
+date<-ymd(substr(args[1], 11, 16));runPath <-paste('/Volumes/NGS/Analysis/covidseq',args[1],sep="/"); head(date)
+
+#=============================================
+ss1 = paste(runPath,"covidseq_output/SampleSheet_Intermediate.csv",sep ="/"); 
+ss2 = paste(runPath,"covidseq_output/lineage/combined_lineage_report.csv",sep ="/"); 
+ss3 = paste(runPath,"covidseq_output/Logs_Intermediates/VirusDetection/kmer/Summary.tsv",sep ="/");
+ss5 = paste(runPath,"covidseq_output/Logs_Intermediates/FastqGeneration/Reports/RunInfo.xml",sep ="/");
```

Comment:
I don't see this variable being used anywhere else in the script; it could probably be removed.
```suggestion

```